apps/guardian-ui/src/languages/en.json

```diff
@@ -55,7 +55,14 @@
       "no-gateways-info-description": "Lightning node operators can connect to your federation to provide Lightning Network interoperability. Once connected, they will appear here."
     },
     "config": {
-      "label": "Federation Config"
+      "label": "Federation Config",
+      "downloadBackup": {
+        "button": "Download Backup",
+        "warningTitle": "Warning",
+        "warningText": "The backup contains private keys and secret material for the federation guardian and must be kept secure.\n Recovery using this backup requires your admin password!",
```

Comment:
didn't test this but from my understanding of the `useTranslation` hook in other projects, line breaks won't work out of the box like this. might need to render it in a `<Trans>` component for this to do what you expect.

Reply:
just removed linebreak, isn't necessary.